lib/hexpm/repository/package_reports.ex

```diff
@@ -16,6 +16,8 @@ defmodule Hexpm.Repository.PackageReports do
       Users.get_by_role("moderator"),
       &email_user_about_new_report(package_report, &1)
     )
+
+    package_report.id
```

Comment:
It is preferable to return the whole struct and then the caller can retrieve the `id` from it instead.